backend/services/implementations/__tests__/testSessionService.test.ts

```diff
@@ -31,4 +32,16 @@ describe("mongo testSessionService", (): void => {
       ...mockTestSession,
     });
   });
+
+  it("deleteTestSession", async () => {
+    const testSession = new testSessionModel(mockTestSession);
+    const savedTestSession = await testSession.save();
+
+    const deletedTestSessionId = await testSessionService.deleteTestSession(savedTestSession.id);
+    expect(deletedTestSessionId).toBe(savedTestSession.id);
+  });
+
+  it("deleteTestSession not found", async () => {
+    expect(testSessionService.deleteTestSession("not_found_id")).rejects.toThrowError();
```

Comment:
Maybe add the exact error message this would throw? 
```
expect(testSessionService.deleteTestSession("not_found_id")).rejects.toThrowError("Test Session id not_found_id not found");
```